### ISSUE TITLE
Release Comment Command

### ISSUE THREAD

**Issue by Mecharyry** [novoda/novoda#48]
## Issue
Currently external collaborators needs access to the `Novoda` Jenkins in order to perform a release 😬 

## Potential Solution
Expose a command that can be executed via GitHub comments for executing a release.

**Comment by ataulm** [comment]
Do we currently allow external collaborators to release? On which projects so far?

**Comment by Mecharyry** [comment]
As far as I know, we grant external collaborators access to Jenkins 🤔  I could be wrong though, @ouchadam any ideas?

**Comment by ouchadam** [comment]
yep, the open source projects are public. I'd prefer to move away from this though as the project repo should contain all the required tools to build itself (minus secrets).

The main benefit would be being able to read the jenkins log for failures like a missing sdk or some other odd error, although an external dev wouldn't be able to fix those issues, only raise them with us
